doc/source/reference/command-line.rst

```diff
@@ -341,19 +341,20 @@ This will print the list of available subcommands:
 
 .. config_help:: yt config
 
-Since the yt version 3.3.2, the previous location of the configuration file
-(``$HOME/.yt/config``) has been deprecated in favor of a location adhering to the
+
+Since yt version 4, the configuration file is located in (``$XDG_HOME_CONFIG/yt/yt.toml``) adhering to the
```

Comment:
wait, did you really change the name of the directory as well ? If so, what's the rationale ? I think the common practice for this is indeed to have `.` prefix for such dirs !

Reply:
In yt 2, the config used to be in `$HOME/.yt/config`.
Since yt 3, the config is in `$XDG_CONFIG_HOME/yt == $HOME/.config/yt`.

This doc update merely reflects the fact the migration will happen from yt 3 to 4 rather than yt 2 to 3.